docs/topics/api/collections.rst

```diff
@@ -27,6 +27,11 @@ The results are sorted by the most recently updated collection first.
     :>json array results: An array of :ref:`collections <collection-detail-object>`.
 
 
+If the ``with_addon`` parameter is passed then :ref:`addons in the collection<collection-addon-detail>` are returned along with the detail.
```

Comment:
In case this helps, I can't think of any situation where we will need addons as part of the result returned from the list API. We do need addons from the detail API.

Reply:
It's easier _not_ to check if it's a detail or list request, tbh, but I could limit this to detail responses to reduce the server load.